pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -26,6 +26,38 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/apis/config"
 )
 
+// ValidateDefaultPreemptionArgs validates that DefaultPreemptionArgs are correct.
+func ValidateDefaultPreemptionArgs(args config.DefaultPreemptionArgs) error {
+	if err := validateMinCandidateNodesPercentage(args.MinCandidateNodesPercentage); err != nil {
+		return err
+	}
+	if err := validateMinCandidateNodesAbsolute(args.MinCandidateNodesAbsolute); err != nil {
+		return err
+	}
+	if args.MinCandidateNodesPercentage == 0 && args.MinCandidateNodesAbsolute == 0 {
+		return fmt.Errorf("both minCandidateNodesPercentage and minCandidateNodesAbsolute cannot be zero")
+	}
+	return nil
+}
+
+// validateMinCandidateNodesPercentage validates that
+// minCandidateNodesPercentage is within the allowed range.
+func validateMinCandidateNodesPercentage(minCandidateNodesPercentage int32) error {
+	if minCandidateNodesPercentage < 0 || minCandidateNodesPercentage > 100 {
```

Comment:
the absolute config parameter has precedence over the percentage one, so one can use `minCandidateNodesPercentage = 0%` to denote that they only want to use an absolute minimum

there's a problem only if both the absolute and the percentage parameter are both zero as that would break things; and we check for that scenario in `ValidateDefaultPreemptionArgs`